### ISSUE TITLE
Recent static builds (x86_64) don't work.

### ISSUE THREAD

**Issue by JakeSFR** [Alexey-Yakovenko/deadbeef#1536]
Devel builds are ok, only '0.7' are affected.
This seems to be the culprit:
```
dlopen error: /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_gui_GTK2.so: undefined symbol: groups_changed
```
Tried in Fatdog64, PCLOS & Mint.

Full log:
```
starting deadbeef 0.7.2-rc1 [static] [portable]
server_start
searching for GUI plugins in /home/testuser/.local/lib64/deadbeef
searching for GUI plugins in /home/testuser/.local/lib/deadbeef
searching for GUI plugins in /home/testuser/deadbeef-0.7.2-rc1/plugins
load_plugin_dir /home/testuser/deadbeef-0.7.2-rc1/plugins: scandir found 53 files
found gui plugin ddb_gui_GTK2.so
added GTK2 gui plugin
found gui plugin ddb_gui_GTK3.so
added GTK3 gui plugin
load gui plugin
checking GUI plugin: GTK2
found selected GUI plugin: GTK2
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_gui_GTK2.so
dlopen error: /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_gui_GTK2.so: undefined symbol: groups_changed
trying /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_gui_GTK2.fallback.so...
selected GUI plugin not found or failed to load, trying to find another GUI plugin
the plugin not found or failed to load
the plugin not found or failed to load
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_gui_GTK2.so
dlopen error: /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_gui_GTK2.so: undefined symbol: groups_changed
trying /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_gui_GTK2.fallback.so...
the plugin not found or failed to load
the plugin not found or failed to load
the plugin not found or failed to load
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_gui_GTK3.so
dlopen error: libgtk-3.so.0: cannot open shared object file: No such file or directory
trying /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_gui_GTK3.fallback.so...
the plugin not found or failed to load
loading plugins from /home/testuser/.local/lib64/deadbeef
loading plugins from /home/testuser/.local/lib/deadbeef
loading plugins from /home/testuser/deadbeef-0.7.2-rc1/plugins
load_plugin_dir /home/testuser/deadbeef-0.7.2-rc1/plugins: scandir found 53 files
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/aac.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/adplug.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/alac.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/alsa.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/artwork.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/cdda.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/converter.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/converter_gtk2.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/converter_gtk3.so
dlopen error: libgtk-3.so.0: cannot open shared object file: No such file or directory
trying /home/testuser/deadbeef-0.7.2-rc1/plugins/converter_gtk3.fallback.so...
plugin not found or failed to load
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/dca.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_ao.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_dumb.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_mono2stereo.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/ddb_shn.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/dsp_libsrc.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/ffap.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/ffmpeg.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/flac.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/gme.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/hotkeys.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/in_sc68.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/lastfm.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/m3u.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/mms.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/mp3.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/musepack.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/notify.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/nullout.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/oss.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/pltbrowser_gtk2.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/pltbrowser_gtk3.so
dlopen error: libgtk-3.so.0: cannot open shared object file: No such file or directory
trying /home/testuser/deadbeef-0.7.2-rc1/plugins/pltbrowser_gtk3.fallback.so...
plugin not found or failed to load
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/pulse.so
dlopen error: libpulse-simple.so.0: cannot open shared object file: No such file or directory
trying /home/testuser/deadbeef-0.7.2-rc1/plugins/pulse.fallback.so...
plugin not found or failed to load
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/shellexec.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/shellexecui_gtk2.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/shellexecui_gtk3.so
dlopen error: libgtk-3.so.0: cannot open shared object file: No such file or directory
trying /home/testuser/deadbeef-0.7.2-rc1/plugins/shellexecui_gtk3.fallback.so...
plugin not found or failed to load
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/sid.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/sndfile.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/supereq.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/tta.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/vfs_curl.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/vfs_zip.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/vorbis.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/vtx.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/wavpack.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/wildmidi.so
loading plugin /home/testuser/deadbeef-0.7.2-rc1/plugins/wma.so
selected output plugin: ALSA output plugin
INFO: loading playlist Default
INFO: from file /home/testuser/.config/deadbeef/playlists/0.dbpl
convgui: gtkui plugin not found
plugin Converter GTK2 UI failed to connect to dependencies, deactivated.
plugin Playlist browser GTK2 failed to connect to dependencies, deactivated.
shellexecui: can't find gtkui plugin
plugin Shellexec GTK2 UI failed to connect to dependencies, deactivated.
gui plugin has quit; waiting for mainloop thread to finish
```

**Comment by Alexey-Yakovenko** [comment]
Thanks!

**Comment by Alexey-Yakovenko** [comment]
fixed in commit #2200c97
